app/com/lucidchart/open/cashy/utils/KrakenClient.scala

```diff
@@ -22,6 +23,9 @@ class KrakenClient {
   protected val apiSecret = configuration.getString("kraken.apiSecret").get
   protected val uploadUrl = configuration.getString("kraken.imageUploadUrl").get
   protected val usageUrl = configuration.getString("kraken.usageUrl").get
+  protected val connectionRequestTimeout = configuration.getInt("kraken.connectionRequestTimeout").get
```

Comment:
Might be nice to use a Duration here so the config can be `5m`. Or at least put the unit in the key `kraken.connectionRequestTimeoutMs`